router/mentors/mentor-router.js

```diff
@@ -0,0 +1,61 @@
+const router = require('express').Router();
+const db = require('./mentor-model');
+middleware = require('./middleware');
+
+
+router.get('/', (req, res) => {
+    db.getMentors()
+    .then(mentors => {
+        res.status(200).json(mentors)
+    })
+    .catch(err => {
+        res.status(500).json(err)
+    })
+});
+
+router.get('/:id', middleware.validateUserId, (req, res) => {
+    const id = req.params.id;
+    db.getMentor(id)
+    .then(id => {
+        res.status(200).json(id)
+    })
+    .catch(err => {
+        res.status(500).json({errorMessage: 'Could not retrieve specified ID'})
+    })
+});
+
+router.post('/', middleware.validateUser, (req, res) => {
```

Comment:
starting here from these routes down they need to have authentication to be added so the routes can only be used if mentor is logged in.

Reply:
Thank you for letting me know